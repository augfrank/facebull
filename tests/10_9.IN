M0   C0   C1  2239
M1   C0   C2  2459
M2   C0   C3  2836
M3   C0   C4  2974
M4   C0   C5  2337
M5   C0   C6  2564
M6   C0   C7  2103
M7   C0   C8  2740
M8   C0   C9  2067
M9   C1   C0  2920
M10   C1   C2  2828
M11   C1   C3  2367
M12   C1   C4  2184
M13   C1   C5  2062
M14   C1   C6  2777
M15   C1   C7  2252
M16   C1   C8  2670
M17   C1   C9  2646
M18   C2   C0  2397
M19   C2   C1  2654
M20   C2   C3  2246
M21   C2   C4  2048
M22   C2   C5  2436
M23   C2   C6  2713
M24   C2   C7  2568
M25   C2   C8  2386
M26   C2   C9  2713
M27   C3   C0  2574
M28   C3   C1  2724
M29   C3   C2  2292
M30   C3   C4  2998
M31   C3   C5  2591
M32   C3   C6  2649
M33   C3   C7  2483
M34   C3   C8  2242
M35   C3   C9  2959
M36   C4   C0  2555
M37   C4   C1  2939
M38   C4   C2  2916
M39   C4   C3  2995
M40   C4   C5  2190
M41   C4   C6  2562
M42   C4   C7  2490
M43   C4   C8  2648
M44   C4   C9  2183
M45   C5   C0  2462
M46   C5   C1  2209
M47   C5   C2  2975
M48   C5   C3  2956
M49   C5   C4  2942
M50   C5   C6  2479
M51   C5   C7  2087
M52   C5   C8  2090
M53   C5   C9  2839
M54   C6   C0  2891
M55   C6   C1  2323
M56   C6   C2  2356
M57   C6   C3  2424
M58   C6   C4  2917
M59   C6   C5  2665
M60   C6   C7  2717
M61   C6   C8  2629
M62   C6   C9  2329
M63   C7   C0  2432
M64   C7   C1  2214
M65   C7   C2  2279
M66   C7   C3  2909
M67   C7   C4  2819
M68   C7   C5  2202
M69   C7   C6  2978
M70   C7   C8  2703
M71   C7   C9  2893
M72   C8   C0  2027
M73   C8   C1  2725
M74   C8   C2  2154
M75   C8   C3  2256
M76   C8   C4  2541
M77   C8   C5  2238
M78   C8   C6  2665
M79   C8   C7  2404
M80   C8   C9  2649
M81   C9   C0  2482
M82   C9   C1  2642
M83   C9   C2  2772
M84   C9   C3  2352
M85   C9   C4  2729
M86   C9   C5  2734
M87   C9   C6  2060
M88   C9   C7  2213
M89   C9   C8  2360
