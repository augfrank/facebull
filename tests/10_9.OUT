21651
0 13 21 34 44 51 57 65 72 87
